Resources/Prototypes/_NF/Entities/Objects/Vehicles/vehicles.yml

```diff
@@ -183,6 +183,166 @@
           params:
             volume: -3
 
+# Enhanced version of the vanilla Janicart that adds a bucket and mop slot.
+- type: entity
+  id: VehicleJanicartNF
```

Comment:
Prefer prefixes to suffixes.  I know we've got both, but prefer prefixes to suffixes.

Reply:
Got it, will adjust that!